.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,44 @@
+name: Tests on Windows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+jobs:
+  unittests:
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+        runner: ["windows.4xlarge"]
+        gpu-arch-type: ["cpu"]
+      include:
+        - python-version: "3.8"
+          runner: windows.8xlarge.nvidia.gpu
+          gpu-arch-type: cuda
+          gpu-arch-version: "11.7"
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/windows_job.yml@main
+    with:
+      repository: pytorch/vision
+      runner: ${{ matrix.runner }}
+      timeout: 120
+      script: |
+        set -euo pipefail
+        
+        # FIXME: Port this to pytorch/test-infra/.github/workflows/windows_job.yml
+        export PATH="/c/Jenkins/Miniconda3/Scripts:${PATH}"
```

Comment:
I need to dig why [putting this into the windows job template](https://github.com/pytorch/test-infra/blob/601c0eb4d962e0cca473e991342bb277bbd4e692/.github/workflows/build_wheels_windows.yml#L95-L97) as I did in pytorch/test-infra#3960 didn't do the job. 

In any case, since this is just fixing the `PATH` for the `conda` binary, this is not blocking for this PR and we can fix later.